### DISCUSSION TITLE
Need an API to find the position of image from a sub-image

### DISCUSSION THREAD

**Discussion by renrenproject** [community/community#110492] [Programming Help]
### Body

Guys I'm implementing a program that can find the position of the target image in a larger image. For example, I input an image(big) and a sub-image(small), like a living room pic and a cat pic, the API will return the cat location axis of living room image to me. I need to save this info into DB.
I'm good at using python, go and C#.
Anyone have recommendations?
Many thanks.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by gbpii** [comment]
A python project: 
https://github.com/jonathanadamrico/FindMe

It works similar, uploading big picture > uploading little object > input threshold value > done.

**Reply by renrenproject** [reply]
Appreciate that, I'll try it.

**Reply by gbpii** [reply]
Tell me if it works as you want, happy coding there!

**Reply by renrenproject** [reply]
Hi gbpii, thanks for catch up, I tried that FindMe but unfortunately the detection not very effective. I gonna consider use keras and train my own model.
Anyway, thanks for help,
